### ISSUE TITLE
[Bug]: Images getting blocked in Events

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/reactsocialnetwork#266]
### Describe the Bug

In the console, while navigated to /events, the images are getting CORS' blocked with a NS_BINDING_ABORTED error, in particular the ones from shutterstock.  We need to implement a method for handling CORS for images people post for their event(s).  We do have CORS package installed on the server.

### Steps to Reproduce

Start the React development server

Go to /events or click on the Events in the SideNav

Inspect the page using the console.

### Which device are you using?

PC Desktop, x64 Core i7 3rd Gen

### Which operating system are you using?

Linux x64, Debian Base

### Which browser are you using?

Firefox 112

### Contribute

- [ ] I am willing to contribute and submit a pull request

**Comment by NigamDevansh** [comment]
@gbowne1 can i work on this issue.

**Comment by gbowne1** [comment]
Ok @NigamDevansh

**Comment by NigamDevansh** [comment]
@gbowne1 so the problem is, we unable to fetch the images of those events which are having shutterstock images as event image url ?
please correct me if i'm wrong.

**Comment by NigamDevansh** [comment]
also when i was going through the /friends route their were two warnings one was "Invalid prop `children` of type `array` supplied to `Panel`, expected `object`." which I removed by changing the propType of children in panel to ".node" from ".object" as`PropTypes.object` expects the `children` prop to be a single React element (e.g., a single JSX element or a component), not an array of elements. When you pass multiple JSX elements as children to a component, React automatically wraps them in an array.
Second error was in friends component the `children prop` was `required so I just commented it out so if needed one can easily uncomment it and use it.
also added the image to the friends list over at card media.

**Comment by gbowne1** [comment]
@NigamDevansh 

Yes, The images are not available because the images are blocked by cors.

The Friends and Panel issues are separate from this issue.

**Comment by NigamDevansh** [comment]
I have created two events one using chrome and one using firefox and both of them are being rendered properly and i have used the shutterstock images in both the events in the image url.
@gbowne1

**Comment by gbowne1** [comment]
@NigamDevansh I opened #275 as a new issue for this.

**Comment by gbowne1** [comment]
You should see the message NS_BINDING_ABORTED in the browser console likely the network tab when on /friends for the images.  This is coming from cors.

**Comment by NigamDevansh** [comment]
can you share a screenshot of the error

**Comment by gbowne1** [comment]
@NigamDevansh 

A screenshot with the Firefox 112 browser console open while navigated to /events

Look at the Network tab "Transferred" column.  Even though the images resulted in a 200 GET it still responded with NS_BINDING_ABORTED.

![Screenshot_2023-07-04_13-19-21](https://github.com/gbowne1/reactsocialnetwork/assets/47549872/58ed5575-45e2-43af-851b-4600173a9fe2)

**Comment by gbowne1** [comment]
Any ideas @manuel12 ?

**Comment by gbowne1** [comment]
Unassigning since the last was in July '23.

**Comment by gbowne1** [comment]
@balajik

**Comment by gbowne1** [comment]
@balajik 

1) I just wondered if this issue still exists
2) I have had a quest to make this application as error free as possible.  
3) I wondered if it was indeed browser behavior.
4) To my knowledge, it does not have any visual impact

**Comment by gbowne1** [comment]
We are using RTL, Jest and Cypress.  Shouldn't be too hard to come up with a test.

**Comment by gbowne1** [comment]
I also was focused on @NigamDevansh's 2nd comment in this thread.

**Comment by gbowne1** [comment]
yes @balajik
